added data policy (#76)
hyphacoop/handbook: data.md

> ## How We Use Data

Reviewer: See other pages

```suggestion
# How We Use Data
```

_The outline and format of this page is adopted from [Projects by IF: How IF uses data](https://www.projectsbyif.com/how-if-uses-data/)._

> 

Reviewer: See: https://raw.githubusercontent.com/hyphacoop/handbook/master/finance.md

```suggestion

### Contents
```

> - [How We Use Data](#how-we-use-data)

Reviewer: ```suggestion
- [About Hypha](#about-hypha)
```

- [Our Websites and Hosted Services](#our-websites-and-hosted-services)
    - [What data we collect](#what-data-we-collect)
    - [Why we collect data](#why-we-collect-data)
    - [Where we keep data](#where-we-keep-data)
    - [How long we keep data](#how-long-we-keep-data)
- [Our Social Media Accounts and Third-party Services](#our-social-media-accounts-and-third-party-services)

> 
> ---

Reviewer: We don't do this line here https://raw.githubusercontent.com/hyphacoop/handbook/master/coronavirus.md

```suggestion
```

> ---

Reviewer: ```suggestion
## About Hypha
```

Reviewer 2: Not the greatest form to use a section header from another page, I think we don't need this as a top level description if it is a graph pointing to our services and domains (maybe that header would be better if there needs to be one)

Reviewer: Yea I was struggling to write a header for this section, but there needs to be something otherwise it looks awkward continuing from the Contents.

>maybe that header would be better if there needs to be one

What do you mean? We put this underneath `## Our Websites and Hosted Services`? That sounds ok to me.

We do our best to protect the personal information you share with us, and do not share them with third parties.
We may however, **display and share anonymized feedback or personal testimonials on our website and social media, or in other publicly viewable places**.

### Where we keep data

Our services are hosted on servers managed by third-party service providers, physically located in Canada.
These servers are accessible to members of Hypha, and specific collaborators as deemed necessary to deliver the services and content.
Where necessary, servers may also be accessed by other parties to comply with laws or to respond to lawful requests and legal process and in an emergency to protect the personal safety of any person.

### How long we keep data

System logs are generally retained for a duration that is appropriate for the particular system.

User accounts necessary to deliver services and content are generally kept indefinitely, and periodically backed up, until the account holders have requested to delete them.
Deleting your account on a particular service generally does not mean all your information is deleted from our periodic backups.

> If you have a concern, please email `data [at] hypha.coop`.

Reviewer 2: look at how we stylize elsewhere, I'm also not particularly worried about emails in plaintext (e.g., data@hypha.coop)

Uploaded presentations are kept for 5 days after you have ended your meeting and log history is kept for 28 days.
These are BigBlueButton's default configurations and automated purging is performed by the software.
Response to polls, raise hand events, and chat messages are cleared after the call has ended.

Our [Jitsi server](https://jitsi.hypha.coop) clears all meeting data in the room after the call has ended.

The [Matrix chat](https://chat.tomesh.net/#/group/+hyphacoop:tomesh.net) in Hypha rooms are periodically wiped to keep only the most recent 3 months of history.
However, as Matrix is a federated chat protocol, chat history older than 3 months may be retained by other homeservers, and in particular, public rooms with users from other homeservers.

## Our Social Media Accounts and Third-party Services

We use several social media accounts to share our work. We occasionally use the analytics tools provided by these platforms.
Our social media accounts include:

- [hyphacoop on Twitter](https://link.hypha.coop/twitter) ([privacy policy](https://twitter.com/en/privacy))

>   <li><a href="https://link.hypha.coop/linkedin" data-proofer-ignore="true" target="_blank">hyphacoop on LinkedIn</a> (<a href="https://www.linkedin.com/legal/privacy-policy" data-proofer-ignore="true" target="_blank">privacy policy</a>)</li>

Reviewer: We cannot use md for the `<li>`?

Reviewer 2: might be because despite being kramdown, gitbook doesn't process md in the same way jekyll does...

should test whether this works correctly regardless, might need whole list in html, sometimes mg doesn't render as expected alongside html

# How We Use Data

> _The outline and format of this page is adopted from [Projects by IF: How IF uses data](https://www.projectsbyif.com/how-if-uses-data/)._

Reviewer 2: ```suggestion
_This page's outline and format is adopted from [Projects by IF: How IF uses data](https://www.projectsbyif.com/how-if-uses-data/)._
```
Would move this below the header matter, typically we've handled crediting not at the top

- [Our Websites and Hosted Services](#our-websites-and-hosted-services)
    - [What data we collect](#what-data-we-collect)
    - [Why we collect data](#why-we-collect-data)
    - [Where we keep data](#where-we-keep-data)
    - [How long we keep data](#how-long-we-keep-data)
- [Our Social Media Accounts and Third-party Services](#our-social-media-accounts-and-third-party-services)

---

Hypha is [a worker co-operative incorporated in Ontario, Canada](https://handbook.hypha.coop/co-operative.html). We run our public website, member handbook, and a variety of services at the domain [hypha.coop](https://hypha.coop) and its subdomains:

- Website: [hypha.coop](https://hypha.coop)
- Handbook: [handbook.hypha.coop](https://handbook.hypha.coop)
- Meetings: [meetings.hypha.coop](https://meetings.hypha.coop)
- Shortlinks: [link.hypha.coop](https://link.hypha.coop)
- Loomio: [loomio.hypha.coop](https://loomio.hypha.coop)
- Nextcloud: [cloud.hypha.coop](https://cloud.hypha.coop)
- BigBlueButton: [bbb.hypha.coop](https://bbb.hypha.coop)
- Jitsi: [jitsi.hypha.coop](https://jitsi.hypha.coop)
- COVID-19 Solidarity: [covid19.hypha.coop](https://covid19.hypha.coop)

> This page was last updated on May 25, 2020. You can see previous versions on [GitHub](https://github.com/hyphacoop/handbook/).

Reviewer 2: Would put crediting line here.

> ## Data Retention Policy

Reviewer: I think this should match index?

```suggestion
## How We Use Data
```

Reviewer: We should also add a section about where data is physically hosted, like in https://www.projectsbyif.com/how-if-uses-data/

> Our BigBlueButton instance keeps your uploaded presentations for 5 days after you have ended your meeting. BigBlueButton log history are kept for 28 days. Currently our BigBlueButton server is not backed up.

Reviewer 2: Maybe we should add a link to the instance here?

Also, to be clear maybe we say something like:
```suggestion
 Currently data from our BigBlueButton is not backed up to another server.
```

Reviewer: How about this?

```suggestion
Our [BigBlueButton server](https://bbb.hypha.coop) keeps a database for registered users and room information. This information is not backed up.

Uploaded presentations are kept for 5 days after you have ended your meeting and log history is kept for 28 days. These are BigBlueButton's default configurations and automated purging is performed by the software.
```

Just to confirm, the purge tasks are part of BBB config and not done via a custom script or manually right? Mentioning this may be informative.

Reviewer: Please also review https://bigbluebutton.org/privacy/ to see if we have missed anything.

Author: The purge tasks is part of BBB's builtin script. There are still standard system log but not sure if we should mention that to?

> Our Jitsi instance we keep standard system logs for debugging purpose only.

Reviewer 2: How long?
Are they backed up?
What info might exist in these logs? 
e.g. 
- IP address
- Timestamp of visit
...?

Reviewer: @ASoTNetworks please refer to https://jitsi.org/meet-jit-si-privacy/ to see what is relevant to us here.
